src/lib/services/returns/returns-mapping-service.js

```diff
@@ -35,7 +37,16 @@ const getReturnRequirements = returnsData => {
  */
 const mapReturnDataToModel = (returnData, returnRequirements) => {
   const externalId = getReturnRequirementExternalId(returnData)
-  const returnRequirement = returnRequirements.find((o) => o.externalId === externalId)
+
+  let returnRequirement
+
+  try {
+    returnRequirement = returnRequirements.find((o) => o.externalId === externalId)
+    // returnRequirement = returnRequirements.find((o) => o.externalId === externalId)
```

Comment:
Did you mean to leave this commented-out line here?

Reply:
Doh!